### PR TITLE
Developer Onboarding Simplification

### PR DESCRIPTION
**Pull request by klardotsh**
This is the `forem/forem` side of a general simplification of developer onboarding on bare metal that I've been tinkering with for a bit now, and I'm quite happy with it in general (the `forem-docs` PR will get into the specifics of why).

This terseifies the README a bit by getting rid of redundant instructions, and instead defers to `forem-docs` entirely for developer environment setup (in part because choosing a path is a little bit more involved than would reasonably fit into this README). It also provides an [rtx](https://github.com/jdx/rtx) config file - this is loosely similar to the `.tool-versions` file `asdf` uses, but since I currently see no convincing arguments for supporting `asdf` over `rtx`, and since the `.rtx.toml` format supports far more usecases (if we ever need them) than `.tool-versions`, here we are. Ruby and NodeJS versions are read through `.ruby-version` and `.nvmrc`, respectively, by RTX's stock language support backwards compatibility offerings, meaning we don't need to duplicate that effort here (which is great, because plenty of other tools in the ecosystem read those two files, including scripts in our own repo).

### CODE REVIEW

File: .rtx.toml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[tools]
+yarn = "1"
+postgres = "13.12"
```
Comment (lightalloy) [inline .rtx.toml:3]: Cool that we are specifying the version, 11 or higher was not true - one of the tests (and so, the related functionality) was failing on 11.

Comment (maestromac) [reply .rtx.toml:3]: Not sure how much this matters, but DEV specify it's using `13.11`

**Review by maestromac — APPROVED** [review]
LGTM!

**Review by jaw6 — APPROVED** [review]
Seems good to me :+1:

File: .rtx.toml (hunk 1 above)
Comment (klardotsh) [reply .rtx.toml:3]: I based this version off of whatever I had locally resolved for `postgres:13` docker container image, as pulled by our docker-compose stack, so it seems we run fine on 13.12. I can't think of a recent time a minor version upgrade broke "userspace" when it comes to Postgres - they're pretty good about retaining backwards-compatibility within a major release series.

**Pull request merged**